src/test/java/org/broadinstitute/hellbender/utils/read/GATKReadAdaptersUnitTest.java

```diff
@@ -249,7 +265,9 @@ public void testGetAndSetUnclippedStartAndEnd( final GATKRead read, final int ex
         samWithUnmappedMate.setMateUnmappedFlag(true);
 
         final Read googleReadWithUnmappedMate = basicGoogleGenomicsRead();
-        googleReadWithUnmappedMate.setNextMatePosition(null);
+        Position matePos = new Position();
+        matePos.setReverseStrand(false);
+        googleReadWithUnmappedMate.setNextMatePosition(matePos);
```

Comment:
if you set the next mate position isn't this read no longer a read with an unmapped mate?